docs/how-to/construct-models.md

```diff
@@ -382,25 +382,98 @@ ans =
 
 ### Writing custom plot functions that can be manipulated
 
-
 # Creating new conductances
 
-## Understanding the `conductance` class
+> code testing this functionality exists in `../xolotl/examples/demo_conductance`.
```

Comment:
use !!! Note syntax

see the end of this [file](https://raw.githubusercontent.com/sg-s/xolotl/master/docs/tutorials/first-neuron.md)